infra/modules/azure_postgres_server/variables.tf

```diff
@@ -101,6 +101,30 @@ variable "replica_zone" {
   default     = null
 }
 
+variable "replica_enabled" {
+  type        = bool
+  description = "(Optional) Enables replica for PostgreSQL Flexible Server."
+  default     = false
+}
+
+variable "geo_redundant_backup_enabled" {
+  type        = bool
+  description = "(Optional) Enables geo redundancy for PostgreSQL Flexible Server backups."
+  default     = false
```

Comment:
this is a breaking change if you put 'false' here. should be opt-out to keep consistent behavior

Reply:
suggest the change